contrib/analyze-local-images/README.md

```diff
@@ -9,6 +9,8 @@ There are absolutely no guarantees and it only uses a minimal subset of Clair's
 To install the tool, simply run the following command, with a proper Go environment:
 
     go get -u github.com/coreos/clair/contrib/analyze-local-images
```

Comment:
We could also just change this line to `goinstall` rather than `go get`. It'll pull in the dependencies, right?